src/error-rescue.js

```diff
@@ -135,6 +135,7 @@ module.exports = class KiteErrorRescue {
         delete this.isSidebarOpen;
       }
     });
+    this.URI = URI;
```

Comment:
Does `this.URI` actually get used?

Reply:
Actually not, but it allows an external piece of code to access the text document if needed. I think I can probably get rid of it with no consequences.